scripts/build-wasm.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+THISDIR=$(dirname "$(realpath "$0")")
+. $THISDIR/build-common-container.sh
+
+LIBNAME=frosty_rust.wasm
+WASM_BINDGEN_OUTNAME=frosty_rust
+WEB_OUTDIR=$THISDIR/../frosty_flutter/example/web/pkg
+
+# Build container image with the build context being the parent directory
+TAG=frosty_rust_wasm_build
+$PROGCMD build -f $THISDIR/build_wasm.Dockerfile -t $TAG $THISDIR/.. || exit 1
+
+build () {
```

Comment:
Since there is code duplication with build-linux.sh, `build()` could be moved to build-common.sh.